src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -0,0 +1,295 @@
+                                <Expander.Header>
+                                    <Grid x:Name="PackageGrid"  >
+                                        <DockPanel HorizontalAlignment="Stretch"
+                                                       >
+                                            <StackPanel HorizontalAlignment="Left"
+                                                            Orientation="Horizontal"
+                                                            Margin="10,5,5,7">
+                                                <TextBlock Text="{Binding Path=Model.Name}"
+                                                               ToolTip="{Binding Path=Model.Name}"
+                                                               FontSize="11"
+                                                               Width="150"
+                                                               TextTrimming="CharacterEllipsis"
+                                                               Margin="15,0,10,0"
+                                                               Foreground="#E5E5E5"></TextBlock>
+                                                <TextBlock Text="{Binding Path=Model.VersionName}"
+                                                               MinWidth="50"
+                                                               FontSize="11"
+                                                               Foreground="#E5E5E5"></TextBlock>
+                                                <TextBlock Text="{x:Static p:Resources.InstalledPackageViewPendingInstallButton}"
+                                                               Visibility="{Binding Path=Model.EnableOldMarkedForUnistallState, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}"
+                                                               FontSize="11"
+                                                               FontStyle="Italic"
+                                                               MinWidth="160"
+                                                               TextAlignment="Center"
+                                                               Foreground="Red"></TextBlock>
+                                            </StackPanel>
+                                            <StackPanel HorizontalAlignment="Right" Orientation="Horizontal" Width="Auto">
+                                                <Button Name="MoreButton"
+                                                                Click="MoreButton_OnClick"
+                                                                Width="16"
+                                                                Height="16"
+                                                                Margin="0,0,10,0"
+                                                                Style="{StaticResource FlatButtonStyle}">
```

Comment:
is there a reason you went with flatbuttonstyle that has a grey outline vs the flaticonbuttonstyle that has the blue outline (like other icon buttons)?

Reply:
No reason, I just happen to pick that one. @Jingyi-Wen, what do you think?